src_c/base.c

```diff
@@ -2086,59 +2084,41 @@ MODINIT_DEFINE(base)
                                          NULL,
                                          NULL};
 
-    if (!is_loaded) {
```

Comment:
Why was the !is_loaded guard removed?

Reply:
Because that was a local variable, lol, and was always set to 0 when the init function is called. So the conditional was useless because it will get executed in all cases.